config/jobs-chromeos.yaml

```
@@ -369,6 +369,21 @@ _anchors:
         - collabora-chromeos-kernel
         - media
 
+  watchdog-reset-arm64: &watchdog-reset-arm64-job
```

```suggestion
  watchdog-reset: &watchdog-reset-job
```

This is in no way architecture-dependent, so we can keep the naming arch-agnostic as well (same for the params) and get rid of `watchdog-reset-x86`.

---

Right - will do.